src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
My pb is actually that in my plugins I use the dtkAbstractProcessFactory to reach my processes (for instance if I am in a plugin A and I want a process from a plugin B). 
I end up playing with a medAbstractProcess (I dont use dtkAbstractProcesse, I cant remember why), but I can't set his input.